go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -195,15 +224,21 @@ func (nt *nodeTracker) updateNode(node *v1.Node) {
 		grName = util.GetGatewayRouterFromNode(node.Name)
 		if gwConf.NodePortEnable {
 			for _, ip := range gwConf.IPAddresses {
-				ips = append(ips, ip.IP.String())
+				ips = append(ips, ip.IP)
 			}
 		}
 	}
 
+	chassisID, err = util.ParseNodeChassisIDAnnotation(node)
+	if err != nil {
+		klog.Infof("Node %s has invalid / no ChassisID annotation (probably waiting on initialization): %v", node.Name, err)
```

Comment:
Could it be that `Warningf` is better than `Infof` here?

Reply:
I'm not sure.  I chose `Infof` because this can happen without it being the result of any bug and it's transient.